pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -3869,7 +3869,8 @@ SELECT ‹'yes'› IN (‹'no'›, ‹'maybe'›, ‹'yes'›)
 query T
 SELECT crdb_internal.redactable_sql_constants(e'\r')
 ----
-‹
›
+‹
+›
```

Comment:
this change is odd

Reply:
I either fat fingered this or GoLand auto-adjusted it because of the awkward carriage return. Fixed.